packages/pipeline/src/globals.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'node-redshift';
```

Comment:
Please define types

Reply:
@LogvinovLeon can you perhaps help @tomhschmidt with this. He might be new to TS.

Reply:
I *am* new to TS :)

I tried to include DefinitelyTyped definitions when they were available. node-redshift didn't have one, so I was planning on just running with this for now, as the usage is pretty small. I can write a type definition if we feel we need it.